zh/enable-tidb-cluster-auto-scaling.md

```diff
@@ -102,7 +102,7 @@ spec:
   ......
 ```
 
-## 快速上手
+## 例子
 
 我们将通过以下指令快速部署一个 3 PD、3 TiKV、2 TiDB，并带有监控与弹性伸缩能力的 TiDB 集群。
```

Comment:
Please use `1,2,3...` to mark these steps.

Reply:
updated